assets/js/googlesitekit/datastore/user/authentication.test.js

```diff
@@ -20,6 +20,7 @@
  * Internal dependencies
  */
 import API from 'googlesitekit-api';
+import { PROVISIONING_SCOPE, EDIT_SCOPE } from '../../../modules/analytics/datastore/constants';
```

Comment:
Not a fan of importing constants specific to a module in this general core test. Since this is about testing the selector itself, we can just provide any arbitrary scope here, rather than rely on constants that come from a completely different "package".